trigger/commit_queue.go

```diff
@@ -109,7 +109,7 @@ func (t *commitQueueTriggers) makeData(sub *event.Subscription) (*commonTemplate
 	}
 
 	slackColor := evergreenFailColor
-	if t.data.Status == evergreen.PatchSucceeded || t.data.Status == evergreen.MergeTestStarted {
+	if t.data.Status == evergreen.LegacyPatchSucceeded || t.data.Status == evergreen.MergeTestStarted {
```

Comment:
We'd want to use `IsSuccessfulVersionStatus` here right?

Reply:
good catch